javascripts/2048.js

```diff
@@ -41,18 +47,15 @@ function newTile(openCells, usedCells) {
   $(tile).attr("data-val", initNum);
   $(tile).text(initNum);
   occupyCell(openCells, usedCells, cellSpace);
-  // emptyCell(openCells, usedCells, cellSpace)
-
 }
 
 function occupyCell(openCells, usedCells, cellSpace) {
   var index = findIndex(openCells, cellSpace);
   openCells.splice(index, 1);
-  var cellExists = findIndex(usedCells, cellSpace)
+  var cellExists = findIndex(usedCells, cellSpace);
   if (cellExists == undefined) {
```

Comment:
Here's a tricky thing about javascript, `undefined` can actually redefined to mean something. WTF right? To be safe, you should run comparisons like `thing === void 0`. `void 0` is always undefined